Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:18.04
-RUN apt update && \
-    apt -y --no-install-recommends install \
-        python2.7-minimal python-pip python-setuptools python-natsort python-repoze.lru \
-        git language-pack-de rsync && \
-    pip install oerplib
-RUN git clone https://github.com/fau-fablab/oerp-pricelist /oerp-pricelist
-WORKDIR /oerp-pricelist
-RUN pip install --upgrade -r requirements.txt
-COPY config.ini config.ini
+# See https://hub.docker.com/r/frolvlad/alpine-python2/dockerfile
```

Comment:
Please consider the license terms of that source (MIT, which is slightly more restrictive than the Unlicense that we use currently)

Reply:
Good point!

Reply:
I have reproduced the copyright notice within the Dockerfile. Looks to me as if that's the only requirement of the MIT license over Unilicense.